frontend/src/components/navigation/UserModal.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import {
+  Box,
+  Circle,
+  Divider,
+  Flex,
+  Heading,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Text,
+} from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { BiUserCircle } from "react-icons/bi";
+import Logout from "../auth/Logout";
+
+type UserModalProps = {
+  showUser: boolean;
+  onClose: () => void;
+  id: string;
+  firstName: string;
+  lastName: string;
+  role: string;
+  isReviewer: boolean;
+};
+
+const UserModal = ({
+  showUser,
+  onClose,
+  id,
+  firstName,
+  lastName,
+  role,
+  isReviewer,
+}: UserModalProps) => {
+  const onShowProfileClick = () => {
+    window.location.href = `/user/${id}`;
+  };
+
+  return (
+    <Modal
+      isOpen={showUser}
+      onClose={onClose}
+      motionPreset="slideInBottom"
+      size="xs"
+    >
+      <ModalOverlay />
+      <ModalContent marginTop="30vh">
+        <Box
+          backgroundColor="blue.50"
```

Comment:
nit: alphabetize these props

Reply:
a few other components have this nit too

Reply:
Just to confirm, which props should I be alphabetizing? The UserModalProps or the ones in the tags?

Reply:
Nevermind I think I know, is it the import props?

Reply:
all the chakra ui props for these components, eg this is sorted alphabetically
```
 <Circle
              backgroundColor="white"
              height="60px"
              marginTop="35px"
              width="60px"
            >
```
(sorry - shouldve mentioned sorting!)